gym/wrappers/time_limit.py

```diff
@@ -28,6 +35,15 @@ def __init__(self, env, max_episode_steps: Optional[int] = None):
         self._elapsed_steps = None
 
     def step(self, action):
+        """Steps through the environment and if the number of steps elapsed >= max episode steps then terminates.
+
+        Args:
+            action: The environment step action
+
+        Returns:
+            The environment step information with the information and done updated
+            if the number of steps elapsed >= max episode steps
+        """
```

Comment:
Missing observation and reward, if we are being pedantic

Reply:
Update the whole docstring to include the whole return type